rtl/include/eclhelper.hpp

```diff
@@ -44,8 +44,8 @@ typedef unsigned short UChar;
 
 //Should be incremented whenever the virtuals in the context or a helper are changed, so
 //that a work unit can't be rerun.  Try as hard as possible to retain compatibility.
-#define ACTIVITY_INTERFACE_VERSION      650
-#define MIN_ACTIVITY_INTERFACE_VERSION  650             //minimum value that is compatible with current interface
+#define ACTIVITY_INTERFACE_VERSION      651
+#define MIN_ACTIVITY_INTERFACE_VERSION  651             //minimum value that is compatible with current interface
```

Comment:
This should changed back to 650, but the version on the previous line can stay at 651 - since it has changed.